pkgs/development/python-modules/capstone/default.nix

```diff
@@ -1,21 +1,24 @@
-{ lib, stdenv
-, buildPythonPackage
-, fetchPypi
+{ buildPythonPackage
+, capstone
 , fetchpatch
+, fetchPypi
+, lib
 , setuptools
-, capstone
+, stdenv
```

Comment:
stdenv and lib should be the first in that order.

Reply:
Fixed the order.